content/documentation.rst

```diff
@@ -431,3 +450,18 @@ page.
 
 Known Issues will include a reference to the internal Trac ID which provided
 further details about that issues
+
+Mock ups and Designs
```

Comment:
maybe

`Mock ups and Designs for the public website`

that is, for the other documentation we should never use raw html, since that documentation is designed to be converted into multiple formats HTML / PDF / Windows Help

so rawHTML will not be translated into PDF

is ok to use raw HTML for the website as that will only be HTML

I think that this secition need an update to let people know about the fact that the documentation is not just HTML ... but is ok for the website to be HTML